diff --git a/node_modules/react-native-image-to-pdf/android/build.gradle b/node_modules/react-native-image-to-pdf/android/build.gradle
index c098672..bdeb0c5 100644
--- a/node_modules/react-native-image-to-pdf/android/build.gradle
+++ b/node_modules/react-native-image-to-pdf/android/build.gradle
@@ -1,25 +1,12 @@
-buildscript {
-    repositories {
-        jcenter()
-        google()
-    }
-    dependencies {
-        classpath 'com.android.tools.build:gradle:3.3.2'
-
-        // NOTE: Do not place your application dependencies here; they belong
-        // in the individual module build.gradle files
-    }
-}
-
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 28
-    buildToolsVersion "28.0.3"
+    namespace "com.anyline.RNImageToPDF"
+    compileSdkVersion rootProject.ext.compileSdkVersion
 
     defaultConfig {
-        minSdkVersion 19
-        targetSdkVersion 28
+        minSdkVersion rootProject.ext.minSdkVersion
+        targetSdkVersion rootProject.ext.targetSdkVersion
         versionCode 1
         versionName "1.0"
     }
